client/src/controllers/BreadcrumbsController.ts

```diff
@@ -0,0 +1,132 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class BreadcrumbsController extends Controller {
+  static targets = ['w-breadcrumbs', 'toggle'];
+  private toggleTarget!: HTMLElement;
```

Comment:
```suggestion
  declare toggleTarget: HTMLButtonElement;
```

No need to use private, it's not a private property, use declare instead. 

More about this here. https://stimulus.hotwired.dev/reference/using-typescript